interceptors/client_test.go

```diff
@@ -340,12 +342,12 @@ func (s *ClientInterceptorTestSuite) TestBiStreamingReporting() {
 	require.NoError(s.T(), ss.CloseSend())
 	wg.Wait()
 
-	require.EqualValues(s.T(), count, 100, "Number of received msg on the wire must match")
+	require.EqualValues(s.T(), 100, count, "Number of received msg on the wire must match")
```

Comment:
Put actual and expected values where they should be.

Reply:
Wow, thanks for those detailed comments (: Next time, no need for this it's very clear! 💪🏽